<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QWidget" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuickLOD</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ATi/AMD</string>
    </attribute>
    <widget class="QGroupBox" name="gbox4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Ctrl + Alt + 4</string>
     </property>
     <widget class="QSlider" name="slider_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-108</number>
      </property>
      <property name="maximum">
       <number>108</number>
      </property>
      <property name="sliderPosition">
       <number>60</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="value_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Now</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Alt+4</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbox1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Ctrl + Alt + 1</string>
     </property>
     <widget class="QSlider" name="slider_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-108</number>
      </property>
      <property name="maximum">
       <number>108</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="value_1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Now</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Alt+1</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbox3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Ctrl + Alt + 3</string>
     </property>
     <widget class="QSlider" name="slider_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-108</number>
      </property>
      <property name="maximum">
       <number>108</number>
      </property>
      <property name="sliderPosition">
       <number>40</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="value_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Now</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Alt+3</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbox5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Ctrl + Alt + 5</string>
     </property>
     <widget class="QSlider" name="slider_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-108</number>
      </property>
      <property name="maximum">
       <number>108</number>
      </property>
      <property name="sliderPosition">
       <number>108</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="value_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_5">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Now</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Alt+5</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="gbox2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Ctrl + Alt + 2</string>
     </property>
     <widget class="QSlider" name="slider_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>201</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-108</number>
      </property>
      <property name="maximum">
       <number>108</number>
      </property>
      <property name="sliderPosition">
       <number>12</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
     <widget class="QLabel" name="value_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>--</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Now</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Alt+2</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="value_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current LOD:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>About</string>
    </attribute>
    <widget class="QTableWidget" name="table_AboutModules">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>391</width>
       <height>291</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Module</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Author</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>QuickLOD</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>1.00</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>GENiEBEN</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>Qt Framework</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>5.2</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>Digia Plc and its subsidiaries</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Theme - Dark Fusion</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>-</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>QuantumCD</string>
      </property>
     </item>
    </widget>
    <widget class="QCommandLinkButton" name="github">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Visit Github page of this application</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
